require.js

```diff
@@ -1119,12 +1149,46 @@
     };
 
 
+    Require.parseMJSONDependencies = function parseMJSONDependencies(jsonRoot) {
+
+        // Clear commented require calls
```

Comment:
I don't think this comment belongs here.

Reply:
yep